tavla/app/(admin)/mapper/[id]/page.tsx

```diff
@@ -35,6 +35,24 @@ export async function generateMetadata(props: TProps): Promise<Metadata> {
     }
 }
 
+export type AuthenticatedUser = {
+    uid: UserId
+    email?: string
+}
+
+async function getAuthenticatedUsers(
```

Comment:
Dro funksjonaliteten her ut i en egen funskjon, og opprettet typen `AuthenticatedUser` for å skille fra `UserDB` som ikke har `email`